### PR TITLE
Proofreading the animations.md chapter

### PR DESCRIPTION
**Pull request by chrishtr**

### CODE REVIEW

File: book/animations.md [hunk 1]
```diff
@@ -64,7 +64,7 @@ what `animate` does we can change what animation occurs.
 
 For example, we can fade an element in by smoothly transitioning its
 `opacity` value from 0.1 to 0.999.[^why-not-one] Doing this over 120
-frames (about two seconds) means increasing the opacity by about 0.008
+frames (about four seconds) means increasing the opacity by about 0.008
```
Comment (chrishtr) [inline book/animations.md:67]: because 30fps

File: book/animations.md [hunk 2]
```diff
@@ -459,7 +459,7 @@ For our opacity example, the (key part of) the display list for one frame
 might look like this:
 
 ::: {.example}
-    Blend(alpha=0.112375)
+    Blend(alpha=0.119866666667)
```
Comment (chrishtr) [inline book/animations.md:462]: fade out, so opposite order

File: book/animations.md [hunk 3]
```diff
@@ -990,12 +990,15 @@ JavaScript code uses a linear interpolation (or *easing function*)
 between the old and new values. Real browsers use a non-linear default easing
 function for CSS transitions because it looks better. We'll implement
 a linear easing function for our browser, so it will look identical to
-the JavaScript and subtly different from real browsers.
+the JavaScript and subtly different from real browsers, but you can try
```
Comment (chrishtr) [inline book/animations.md:993]: Not necessary to copy this fix to the printed book.

File: book/animations.md [hunk 4]
```diff
@@ -2344,14 +2356,14 @@ CSS property and parsing of `@keyframe` to implement the demos
...
 
-13-6 *Overlap testing with transform animations*. Our
-browser currently does not overlap test correctly in the presence of transform
-animations that cause overlap to come and go. (You'll need to have already done
-Exercise 13-3.) First create a demo that
-exhibits the bug, and then fix it. One way to fix it is to enter "assume
-overlap mode" whenever an animated transform display item is encountered. This
-means that every subsequent display item is assumed to overlap the animating
-one (even if it doesn't at the moment), and therefore can't merge into any
+13-6 *Overlap testing with transform animations*. (You'll need to have already
```
Comment (chrishtr) [inline book/animations.md:2359]: Just moves the parenthetical to the front.

File: book/animations.md [hunk 5]
```diff
@@ -2372,8 +2384,7 @@ One way to reduce that problem is to stop merging paint chunks that would make
 the total area of the `skia.Surface` larger than some fixed value. Implement
 that.[^tiling-helps]
 
- [^tiling-helps]: Another way is via surface tiling (this technique was briefly
- discussed in a Go Further block in Chapter 11).
+ [^tiling-helps]: Another way is via surface tiling.
```
Comment (chrishtr) [inline book/animations.md:2387]: Looks like we removed that Go Further.

File: book/animations.md [hunk 6]
```diff
@@ -2313,10 +2327,8 @@ raster either.
 
 [tr-example]: examples/example13-transform-transition.html
 
-13-4 *Width animations*. Implement the CSS `width` and `height` properties; when
-`width` is set to some number of pixels on an element, the element should be
-that many pixels wide, regardless of how its width would normally be computed;
-the same goes for `height`. Make them animatable; you'll need a variant of
+13-4 *Width/height animations*. (You'll need to have done Exercise 6-2 first.)
+Make `width` and `height` animatable; you'll need a variant of
```
Comment (pavpanchekha) [inline book/animations.md:2331]: In previous chapters the "do Exercise X-Y first" text comes after the first sentence before the hints, and it's not parenthesized. Can we do that here?

File: book/animations.md (hunk 4 above)
Comment (pavpanchekha) [reply book/animations.md:2359]: Well, see above, we should be consistent where possible

Comment (chrishtr) [reply book/animations.md:2359]: ditto

**Review by pavpanchekha — APPROVED** [review]

**Pull request merged**